build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val wartRemoverSettings = {
       Wart.ArrayEquals,
       Wart.AnyVal,
       Wart.EitherProjectionPartial,
-      Wart.Enumeration,
+      //Wart.Enumeration,
```

Comment:
I actually agree with Wart remover in that I'm not a fan of enumerations

Reply:
ok but Enumeration was a nice easy way to deal with the different statuses and they are easy to serialise